delivery_postlogistics_dangerous_goods/tests/test_postlogistics_dangerous_goods.py

```diff
@@ -30,6 +30,7 @@ def setUpClassWebservice(cls):
         cls.service_class = PostlogisticsWebServiceDangerousGoods(
             cls.env.user.company_id
         )
+        return None
```

Comment:
Isn't it the same without it?

Reply:
Return will be needed as we are calling super, else pre-commit will fail.
So, we return the super result now.